### PR TITLE
add some log for kubectl wait

### PR DESCRIPTION
**Pull request by CaoDonghui123**
#### What type of PR is this?

/kind feature
#### What this PR does / why we need it:
add  `See 'kubectl wait -h' for help and examples` for `kubectl wait` error.
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go [hunk 1]
```diff
@@ -155,7 +155,7 @@ func (flags *WaitFlags) ToOptions(args []string) (*WaitOptions, error) {
 	if err != nil {
 		return nil, err
 	}
-	conditionFn, err := conditionFuncFor(flags.ForCondition, flags.ErrOut)
+	conditionFn, err := conditionFuncFor(cmd,flags.ForCondition, flags.ErrOut)
```
Comment (dougsland) [inline staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go:158]: I would add a space between cmd and flags

"cmd, flags"

File: staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go [hunk 2]
```diff
@@ -198,7 +198,7 @@ func conditionFuncFor(condition string, errOut io.Writer) (ConditionFunc, error)
 		}.IsConditionMet, nil
 	}
 
-	return nil, fmt.Errorf("unrecognized condition: %q", condition)
+	return nil,cmdutil.UsageErrorf(cmd,"unrecognized condition: %q", condition)
```
Comment (dougsland) [inline staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go:201]: the same apply here

**Review by dougsland — COMMENTED** [review]
LGTM,  comments regarding readability only.

File: staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go (hunk 2 above)
Comment (CaoDonghui123) [reply staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go]: Thank you for your help

**Pull request closed without merging**